Feature/password reset 2fa (#763)
MapsPeople/docs: src/web/v4/guides/authentication/2fa.md

---
title: 2-Factor Authentication
eleventyNavigation:
    key: 2fa
    parent: various
    title: 2-Factor Authentication
    order: 440
---

## Enabling 2-Factor Authentication

> Enabling 2-Factor Authentication (2FA) is done by the user themselves via the CMS. If a user clicks their avatar in the top-right corner, a drop-down menu will appear, where "Settings" should be clicked. The user should now follow the instructions to enable 2FA.

Reviewer: ```suggestion
You can enable 2-Factor Authentication (2FA) in the MapsIndoors CMS. Click the avatar in the top-right corner, and click on "Settings". Follow the instructions to enable 2FA.
```

We should talk to the user, not about them. It makes the sentences shorter and it's easier to write in the [active voice](https://www.grammarly.com/blog/active-vs-passive-voice/) when addressing someone directly.

![2fa instructions](/assets/various/Dashboard/2fa.png)

## Disabling 2FA

> Once 2FA is enabled, this page will change to allow disabling of 2FA, should the user want to do this at a later point. Simply follow the instructions on the page.

Reviewer: ```suggestion
After you've enabled 2FA, the page will show instructions on how to disable 2FA again.
```

![2fa-disable instructions](/assets/various/Dashboard/2fa-disable.png)

> If you, or a user, loses access to your account and are unable to use 2FA for any reason, please contact your representative at MapsPeople, and we will work with you to find a solution.

Reviewer: ```suggestion
If you lose access to your account and are unable to use 2FA for any reason, please [get in touch](https://resources.mapspeople.com/contact-us), and we'll do our best to find a solution.
```

Reviewer: Whenever there's a call-to-action on getting in touch, I try to include a link to the page on our website or Support Portal.

---
title: 2-Factor Authentication
eleventyNavigation:
    key: 2fa
    parent: various
    title: 2-Factor Authentication
    order: 440
---

## Enabling 2-Factor Authentication

Enabling 2-Factor Authentication (2FA) is done by the user themselves via the CMS. If a user clicks their avatar in the top-right corner, a drop-down menu will appear, where "Settings" should be clicked. The user should now follow the instructions to enable 2FA.

![2fa instructions](/assets/various/Dashboard/2fa.png)

## Disabling 2FA

Once 2FA is enabled, this page will change to allow disabling of 2FA, should the user want to do this at a later point. Simply follow the instructions on the page.

![2fa-disable instructions](/assets/various/Dashboard/2fa-disable.png)

> ### Disabling 2FA on behalf of a user

Reviewer 2: I don't think this should be in the public doc. At the most it should say something about contacting someone at MapsPeople in order to help disabling 2FA.

Reviewer 2: @JG1995

Author: I can remove the second part about if the user loses access, and just ask them to defer to us, but the part about the user self-disabling is literally in the CMS, so I feel like it would be weird not to mention it.

Reviewer 2: I was only referring to the section `### Disabling 2FA on behalf of a user`.

Author: Ah alright, the other part was just included in the snippet GitHub highlighted, will push a fix in a moment
